README.md

```diff
@@ -1 +1,52 @@
-# crypto-market-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Crypto Market App
+
+> Crypto Market is a SPA that allow users to check live prices of cryptocurrencies. Built with React and CoinGecko API.
+
+## Built With
+
+- React
+- Redux
+
+## Live Demo
+
+-
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Install git in your computer.
+- Install a code editor.
+- A web browser to view the output.
+
+### Setup
+
+- To clone my repository run this command `git clone https://github.com/karayamanemre/crypto-market-app.git`
+- Run `npm install` to install the dependencies.
+- Run `npm start` to start the app.
+
+## Authors
+
+👤 **Emre Karayaman**
+
+- GitHub: [@karayamanemre](https://github.com/karayamanemre)
+- Twitter: [@KarayamanEmre](https://twitter.com/KarayamanEmre)
+- LinkedIn: [Emre Karayaman](https://www.linkedin.com/in/emre-karayaman-a7b45b243/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
```

Comment:
- [ ] Towards the end of your README there should be an "Acknowledgement" section that includes credit to Nelson Sakwa whose design was used.